### PR TITLE
Diagnostics now returns a 204 No Content.

### PR DESCRIPTION
**Pull request by gklopper**
Slightly smaller than a 1 px img.

And probably more correct.

### CODE REVIEW

File: diagnostics/app/controllers/Application.scala [hunk 1]
```diff
@@ -1,9 +0,0 @@
-package controllers
-
-import common.Logging
-import play.api.mvc._
-
-object Application extends Controller with Logging {
```
Comment (gklopper) [inline diagnostics/app/controllers/Application.scala]: Shoud have got rid of this when we got rid of the Guardian Management library

File: diagnostics/app/controllers/TinyResponse.scala [hunk 2]
```diff
@@ -0,0 +1,8 @@
+package controllers
+
+import model.NoCache
+import play.api.mvc.{Result, Results}
+
+object TinyResponse extends Results {
```
Comment (gklopper) [inline diagnostics/app/controllers/TinyResponse.scala]: Maybe just inline `NoCache(NoContent)`

**Comment by phamann** [comment]
We also need to allow POST requests on all of these endpoint so that we can start using the [Beacon API](http://www.w3.org/TR/beacon/)

**Comment by phamann** [comment]
:+1: Ship it!

**Pull request merged**